UnityProject/Assets/Scripts/Debug/Logger.cs

```diff
@@ -130,6 +130,8 @@ private static void TryLog(string message, LogLevel messageLevel, Category categ
 			return;
 		}
 
+		LogTraceFormat("Message {messageLevel} {0} did not specify a category!", Category.CodeQuality);
```

Comment:
it's not a $ string so {messageLevel} wouldn't work
and you didn't pass any arguments for {0} to work either

Reply:
You're right. When I fixed that more stuff went wrong, so I decided against doing it.